src/views/portfolio/vulnerabilities/AffectedProjects.vue

```diff
@@ -25,9 +26,20 @@
             title: this.$t('message.name'),
             field: "name",
             sortable: true,
-            formatter(value, row, index) {
-              let url = xssFilters.uriInUnQuotedAttr("../../../projects/" + row.uuid);
-              return `<a href="${url}">${xssFilters.inHTMLData(value)}</a>`;
+            formatter: (value, row, index) => {
+              const url = this.$router.resolve({name: 'Project Finding Lookup',
+                  params: {'uuid': row.uuid, affectedComponent: row.affectedComponentUuids[0],
+                      vulnerability:this.vulnerability}}).href;
```

Comment:
This is a seriously cool feature!

However, it suffers from the same limitation as the link to the dependency graph did before: If more than one component is vulnerable, the user is only taken to a single occurrence of the finding, which can be confusing.

Perhaps this button should only include the vulnerability UUID, and not the other parts of the matrix? This way, all occurrences of the vulnerability would be shown.

Reply:
I removed the limitation as suggested. Is it ok if only the first entry opens?

Reply:
Yeah that should be fine.